src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,101 @@
+.product-card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
```

Comment:
Consider using SCSS variables for colors and other repeated values. This will make it easier to maintain and update your styles. Create a separate file for SCSS variables as required by the task.